doc/source/whatsnew/v1.3.5.rst

```diff
@@ -28,7 +28,7 @@ Fixed regressions
 
 Bug fixes
 ~~~~~~~~~
--
+- Fix a bug where :class:`ExtensionDtype` and :class:`ExtensionArray` subclasses were not (de)serialized for ``orient='table'`` (:issue:`20612`, :issue:`44705`).
```

Comment:
Can you place this in `whatsnew/1.4.0.rst` and place it under the "new features" section, as it really is more of a new feature than fixing a bug?

Reply:
For sure, will do!